drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -622,8 +623,8 @@ static int stop_gps(struct device *dev, bool is_timeout)
 	retval = nrf_setsockopt(drv_data->socket,
 				NRF_SOL_GNSS,
 				NRF_SO_GNSS_STOP,
-				NULL,
-				0);
+				&delete_mask,
+				sizeof(nrf_gnss_delete_mask_t));
```

Comment:
How can I know if these are valid parameters for the `NRF_SO_GNSS_STOP` option when the documentation still says this option takes no payload?
It seems that such change should be rather applied together with the corresponding update of `nfrxlib` revision.

Reply:
Fair comment, but it doesn't change the fact that in this same driver also NRF_SO_GNSS_START takes nrf_gnss_delete_mask_t as payload, while documentation says there is no payload. So situation isn't perfect now and it doesn't get much worse with this change. 
We'll make a new update of nrfxlib soon anyway. This task is preparation for that and helps testing the new lib if gps driver side is already future-proof.

But, it's up to you to merge or wait with this.

Reply:
I think that we should wait then and merge this change together with the update of nrfxlib.

Reply:
After a week I think now that this should be merged, because lack of this task really complicates ongoing testing of future bsdlib releases.
So could you please merge this now, there is no real reason to not do it. The harm is bigger if not done. Thanks!